presentations/2025-06-02_replaymeeting.py

```
@@ -0,0 +1,176 @@
+    "JB026",  # WT
+    "JB030",  # WT
+    "JB033",  # WT
+]
+
+
+def already_done_grosmark(session: Any, rewarded: bool) -> bool:
+    results_path = Path(
+        "/Users/jamesrowland/Code/viral/results/pairwise-reactivations-ITI"
+    )
+
+    for file in results_path.glob(f"*.npy"):
```

Ruff (0.14.4)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)